### DISCUSSION TITLE
Random crashing when using an umbrella Feed

### DISCUSSION THREAD

**Discussion by upsilondynamics** [unoplatform/uno#9277] [General]
We have an application that crashes consistently. No errors are thrown, no exceptions can be caught to identify the source or the problem. 

It seems to happen when we try to consume a web API Via an umbrella feed. 

Any idea or how we can fix this when we can’t debug the issue?

**Comment by francoistanguay** [comment]
We'll need more info and a sample repro in order to be able to help.
